broker/providers/ansible_tower.py

```diff
@@ -161,6 +161,37 @@ def get_awxkit_and_uname(
     awxkit_config.base_url = url
     if root is None:  # support mock stub for unit tests
         root = awxkit.api.Api()
+
+    temp_token_desc = None
+
+    # If no token was provided, try to create a temporary token
+    if not token and uname and pword:
+        helpers.emit(auth_type="password")
+        logger.warning(
+            "You should be using token-based authentication.\n"
+            "I will attempt to create and use a temporary token."
+        )
```

Comment:
I would prefer broker to be impersonal. 
```suggestion
        logger.warning(
            "Token-based authentication should be used.\n"
            "An attempt will be made to create and use a temporary token."
        )
```